### PR TITLE
added kicad project for hotswap board based on the work of weteor

### PR DESCRIPTION
**Pull request by dreipunkteinsvier**
I added the kicad project for the hotswap version of the pcb. Placed the files next to your original solder pcb.
Feel free to move elsewhere. The main readme should be extended with the info, but I'll leave that up to you. <3

### CODE REVIEW

File: kicad/le_chiff_ble_hotswap/fp-info-cache
Comment (MangoIV) [inline kicad/le_chiff_ble_hotswap/fp-info-cache:1]: Die Datei brauch mer nich https://forum.kicad.info/t/fp-info-cache-purpose/16569

File: kicad/le_chiff_ble_hotswap/le_chiff_ble_hotswap.kicad_prl
Comment (MangoIV) [inline kicad/le_chiff_ble_hotswap/le_chiff_ble_hotswap.kicad_prl:1]: Die hier ooch nich

**Review by MangoIV — APPROVED** [review]
Die beiden files noch wegmachen (am besten force push, damit ich nicht squashen muss) und dann passt das ;)

**Comment by dreipunkteinsvier** [comment]
Ärrledschd

**Pull request merged**